### ISSUE TITLE
savings should not be greater than costs

### ISSUE THREAD

**Issue by jessegoodier** [kubecost/cost-analyzer-helm-chart#1738]
**Describe the bug**  

I have seen this on multiple clusters, including in hosted.
https://demo.kubecost.xyz/overview for example

In any case, it looks really bad and we lose loads of credibility on our home screen.

**To Reproduce**  

I'm not 100%- likely only happens when cloud integration is enabled (which should be almost 100%)

**Expected behavior**  

At a minimum, hide the savings box if the savings>costs
Ideally we could have a better calculation.

**Screenshots**  

![image](https://user-images.githubusercontent.com/31039225/195459347-a1e03a21-39e0-493f-84ce-cf6f5275dc88.png)
![image](https://user-images.githubusercontent.com/31039225/195459402-20852249-785f-42e2-81d3-ac6d582990c8.png)
![image](https://user-images.githubusercontent.com/31039225/195459611-262a09ea-8720-4aaf-925e-33ae03adc388.png)
![image](https://user-images.githubusercontent.com/31039225/195459963-04067b94-f3f9-4531-9ec4-b44fabab04c2.png)

**Collect logs (please complete the following information):**

Please let me know if you would like cluster access/logs

**Comment by dwbrown2** [comment]
Jesse, thanks for reporting and agree with your view on impact. My assessment is that in this case we're double counting pods as 1) abandoned and should be deleted and 2) over-provisioned and should be right-sized. Is that consistent with your understanding? We should have a heuristic to prevent this. 

That said, when I look at https://demo.kubecost.xyz/overview (which has a cloud integration completed) it's actually expected that Savings > K8s costs given that there are external cloud savings are present. 

<img width="568" alt="image" src="https://user-images.githubusercontent.com/298359/195490289-4a99f234-75c5-41c1-9e32-ad53eaa0eb91.png">

**Comment by jessegoodier** [comment]
For demo.kubecost.xyz, I agree, that's what is happening.

In the screenshot with the $297k abandoned workloads: perhaps that is the logic error Ajay just fixed. In that cluster, "Delete unassigned resources" is $0.

If the savings identified card had a different title, that would go a long way to addressing the confusion. 
"Total possible savings (including cloud resources)"
Which is a bit wordy.

Totally open to solutions here.

**Comment by jessegoodier** [comment]
I think the spacing in the new overview page, with the new wording, addresses the main concern. Closing-
